server/controllers/LoanController.js

```diff
@@ -70,10 +72,10 @@ const loanController = {
           });
 
         } 
-        statusMessageFunction(res, 400, `Oops!! You have unpaid Loan of ## ${haveApplyLoan.rows[0].balance} ## applied on ${haveApplyLoan.rows[0].created_on} , Please repay the loan!` ) 
+        statusMessageFunction(res, 400, `You have unpaid Loan!` ) 
       } 
       else {
-        statusMessageFunction(res, 400, 'Sorry! Your are not yet verified, Please contact Admin !' )
+        statusMessageFunction(res, 401, 'You are not yet verified !' )
```

Comment:
By the time a user accesses this route, they are already authenticated so no need of a 401 , use 403 for forbidden